src/main/java/ir/ui/golestan/data/repository/ScoreRepository.java

```diff
@@ -13,6 +13,5 @@
 public interface ScoreRepository extends JpaRepository<Score, Score.ScoreId> {
     @Query("SELECT s FROM Score s WHERE s.id = :id")
     Collection<Score> findScoresById(@Param("id") int id);
```

Comment:
```suggestion
    Collection<Score> findScoresBystudentId(int studentId);
```
intellij nadarm. VScode daram mizanam. Alan injuri okye?
badam tu StudentController seda mizanam ba @requestEntity o auth.
@sbaneshi